Applications/TransferLearning/Draw_Classification/jni/main.cpp

```diff
@@ -440,12 +459,22 @@ int main(int argc, char *argv[]) {
   }
 
   /** Do the training */
-  status = trainModel(config.c_str());
+  try {
+    status = trainModel(config.c_str());
+  } catch (...) {
+    std::cerr << "Failed train model\n";
+    return 1;
```

Comment:
let's not exist here. Please reset the `feature_state` back to `NOT_CHECKED_YET` if the `__TIZEN__` is defined.
This is a bug in the existing code as well in line 458.

Reply:
You mean no exit and just reset the feature_state?

Reply:
Exit is fine here, but reset the feature_state before exit.

Reply:
Done